src/Json.Schema.Validation.Cli/Json.Schema.Validation.Cli.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Json.Schema.Validation</id>
+    <version>$version$</version>
+    <title>Microsoft Json Schema Validation</title>
+    <authors>microsoft</authors>
+    <owners>microsoft,tse-securitytools</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>A utility that validates files according to a JSON schema.</description>
+    <releaseNotes>Version $version$ of the JSON validation utility (for SARIF v2.0.0)</releaseNotes>
```

Comment:
>for SARIF v2.0.0 [](start = 68, length = 16)

There is slightly more justification for this phrase here than there was in the ToDotNet .nuspec file, but I would still omit it. This tool emits its _output_ in SARIF v2.0. Is that what you meant by this phrase?